packages/gatsby/src/utils/wait-until-jobs-complete.ts

```diff
@@ -2,7 +2,7 @@ import { emitter, store } from "../redux"
 
 import { waitUntilAllJobsComplete as waitUntilAllJobsV2Complete } from "./jobs-manager"
 
-export const waitUntilAllJobsComplete = (): Promise<void> => {
+export const waitUntilAllJobsComplete = (): Promise<any> => {
```

Comment:
This is preparing for follow up PRs where returned value is actually meaningful (hence no `void`, but also not concrete types just yet?

Reply:
Yes, we're now returning the results of the jobs api calls